arch/risc-v/include/common/syscall.h

```diff
@@ -1,5 +1,5 @@
 /****************************************************************************
- * arch/risc-v/include/rv64gc/syscall.h
+ * arch/risc-v/include/common/syscall.h
```

Comment:
let's move to the up level instead common folder.

Reply:
Done